### PR TITLE
[Configuration] nullify empty recruitmentTarget value

### PR DESCRIPTION
**Pull request by llevitis**
This PR fixes #3071 and addresses Redmine ticket 13057. If a new project name is submitted with an empty Recruitment Target value, the submission will successfully be saved with a null value for Recruitment Target instead of returning the error message "Failed to save, same name already exists".

### CODE REVIEW

File: modules/configuration/ajax/updateProject.php [hunk 1]
```diff
@@ -26,15 +26,18 @@
 $db          = $factory->database();
 $ProjectList = Utility::getProjectList();
 
-// if a new project is created add the new project.
+// if a new project is created add the new project.php
 // Otherwise, update the existing project.
 if ($_POST['ProjectID'] === 'new') {
     if (!in_array($_POST['Name'], $ProjectList) && !empty($_POST['Name'])) {
+        $recTarget
```
Comment (kongtiaowang) [inline modules/configuration/ajax/updateProject.php]: Could u put line 33-35 outside the if loop, then you don't need to write twice.

**Comment by kongtiaowang** [comment]
LGTM

**Comment by ridz1208** [comment]
@kongtiaowang did you test it ?

**Comment by kongtiaowang** [comment]
@llevitis 
It needs to modify updateSubproject.php.

**Comment by kongtiaowang** [comment]
LGTM

**Pull request merged**